cartographer_ros/cartographer_ros/node.cc

```diff
@@ -179,7 +179,7 @@ void Node::Initialize() {
       &Node::PublishSubmapList, this));
   wall_timers_.push_back(node_handle_.createWallTimer(
       ::ros::WallDuration(options_.pose_publish_period_sec),
```

Comment:
Should `pose_publish_period_sec` be renamed, now that the function talks about 'trajectory states'?